dracclient/resources/raid.py

```diff
@@ -259,6 +259,11 @@ def _parse_drac_physical_disk(self,
                                                        uri)
         drac_bus_protocol = self._get_physical_disk_attr(drac_disk,
                                                          'BusProtocol', uri)
+        if uri == uris.DCIM_PCIeSSDView:
```

Comment:
Instead of having this "if" statement, can you just pass "allow_missing=True" to _get_physical_disk_attr() below?  That should make it so that you don't need the "if".

Reply:
Hey Chris,
The reason i added a condition above  at line-262  ` if uri == uris.DCIM_PCIeSSDView:` is because if there uri which has only physical disks and not nvme drives then ``bus``  id won't be present in that response. Even if i mention `allow_missing=True` in below  at line 293. I will get Attribute error which will say : `Attribute 'Bus' is missing from the response` in above line.
We will receive Bus id only when there is uri :  uris.DCIM_PCIeSSDView . And there is not `Bus` id  in response when uri is `uris.DCIM_PhysicalDiskView`.
 I think other way we can use is like this below:

` bus = self._get_physical_disk_attr(drac_disk,
                                           'Bus', uri,
                                           allow_missing=True)
        if bus is not None:
            bus = bus.upper()
        else:
            bus = None

        return PhysicalDisk(
            id=..
            description=..
            ..
            ..
            bus=bus)

Please let me know your thought on this.`

Reply:
yes, the above code looks good to me

Reply:
Thanks